docs/index.html

```diff
@@ -41,7 +41,8 @@ <h1><a href="index.html" id="logo">Constellation</a></h1>
                         <li><a href="#banner" class="scrolly">About</a></li>
                         <li><a href="#features" class="scrolly">Features</a></li>
                         <li><a href="download.html" class="scrolly">Download</a></li>
-                        <li><a href="blog.html" class="scrolly">Blog</a></li>
+			                  <li><a href="blog.html" class="scrolly">Blog</a></li>
+                        <li><a href="\constellation-app.github.io\help\constellation\CoreFunctionality\src\au\gov\asd\tac\constellation\functionality\docs\about-constellation.html" class="scrolly">Documentation</a></li>
```

Comment:
I've changed the relevant links to the above - how come `//` is needed at the start instead of just `/`
Is that a URL specific feature?

Reply:
@aldebaran30701 so `/` it would be a relative url on a site. Using `//` is a shortcut to continue using `https://` or `http://` depending on what the website is loaded as.

I am not sure how the leading `\` is being accepted on Windows and how the url with backslash is working. They should all be forward slashes right?